Patches/Xenn/ThingDefs_Races/Race_Xenn.xml

```diff
@@ -4,121 +4,125 @@
 	<Operation Class="PatchOperationSequence">
 		<success>Always</success>
 		<operations>
-	<!-- ========== Look for Fido ========== -->
-	<li Class="CombatExtended.PatchOperationFindMod">
+			<!-- ========== Look for Fido ========== -->
+			<li Class="CombatExtended.PatchOperationFindMod">
 				<modName>Xenn Race</modName>
-	</li>
-	<!-- ========== Found Fido ========== -->
-	
-	<!-- ========== Turn our dog boys into dog *boys* ========== -->
-	<!-- ========== To be human is to be a tall brick ========== -->
-	<li Class="PatchOperationAddModExtension">
-	<!-- === Note to future furball compatibility nerds: 		=== -->
-	<!-- === Paste the defName of your horrible alien there   |	=== -->
-	<!-- === 												  V	=== -->
-		<xpath>*/AlienRace.ThingDef_AlienRace[defName="Alien_Xenn"]</xpath>
-		<value>
-			<li Class="CombatExtended.RacePropertiesExtensionCE">
-				<bodyShape>Humanoid</bodyShape>
 			</li>
-		</value>
-	</li>
+			<!-- ========== Found Fido ========== -->
 
-	<!-- ========== Gives Gun Gizmos to Xenn HUD thing ========== -->
-	<!-- ========== Also Makes Em Suppressable		   ========== -->
-	
-	<li Class="PatchOperationAdd">
-	<!-- === Shouldn't need to change this at all		  === -->
-	<!-- === Unless you changed the name of your BaseXennPawn === -->
-	<!-- === In which case change "BaseXennPawn" to that name === -->
-		<xpath>*/ThingDef[@Name="BaseXennPawn"]/comps</xpath>
-		<value>
-			<li>
-				<compClass>CombatExtended.CompPawnGizmo</compClass>
+			<!-- ========== Turn our dog boys into dog *boys* ========== -->
+			<!-- ========== To be human is to be a tall brick ========== -->
+			<li Class="PatchOperationAddModExtension">
+				<!-- === Note to future furball compatibility nerds: 		=== -->
+				<!-- === Paste the defName of your horrible alien there   |	=== -->
+				<!-- === 												  V	=== -->
+				<xpath>*/AlienRace.ThingDef_AlienRace[defName="Alien_Xenn"]</xpath>

```

Comment:
In all cases, please replace `<xpath>*/ ...` with `<xpath>/Defs/ ...` as  */ loads far slower.  `<xpath>Defs/ ...` is also acceptable to be left in place.

Reply:
I'll change those to `Defs/` ASAP. I've never really liked `*/` anyway, for the exact reason you stated.

Reply:
Fixed in ce3d2f3